app/views/conversations/show.html.erb

```diff
@@ -1,15 +1,100 @@
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+
+        <div class="text-white text-opacity-40 text-xs mx-2 mt-10 mb-2">
+          Yesterday
+        </div>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+        <%= render 'conversation_sidebar', title: "Custom opacity in tailwind" %>
+      </div>
+
+      <div id="left-footer" class="flex-none flex text-white mb-4 mt-3">
+        <div class="h-10 flex pr-2 rounded-lg hover:bg-zinc-400 hover:bg-opacity-25 justify-between items-center w-full group cursor-pointer mb-1">
+          <div id="profile" class="ml-2 mr-2 w-7 h-7 bg-white rounded-full"></div>
+          <div class="flex-1 text-white truncate">
+            Keith Schacht
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+  <div id="right-column" class="flex flex-1 flex-col h-screen bg-white dark:bg-graphite">
+    <!-- Header that does not grow -->
+    <div id="wide-header" class="flex-none justify-between ml-2 mt-2 mb-3 h-10 text-black dark:text-white text-lg hidden md:flex">
+      <%= render 'gpt_header', conversation: @conversation %>
+
+      <div id="right-aligned-content" class="hover:bg-gray-100 cursor-pointer inline-flex mr-5 border border-gray-300 rounded-lg px-2 py-2 mt-1 tooltip tooltip-bottom hover:tooltip-open" data-tip="Share">
+        <%= icon "arrow-up-tray", variant: :outline, size: 18, class: 'text-black' %>
+      </div>
+    </div>
+
+    <div id="narrow-header" class="flex-none h-14 text-lg text-center bg-white dark:bg-zinc-800 md:hidden">
+      <div class="flex-1 mb-2 border-b border-zinc-300 dark:border-zinc-600 text-black dark:text-white dark:bg-graphite flex items-center justify-center">
+        <%= render 'gpt_header', conversation: @conversation %>
+      </div>
+    </div>
+
+    <div id="right-content-container" class="flex flex-col flex-grow overflow-auto">
+      <div id="right-content" class="flex-grow overflow-auto">
+        <div class="w-full md:max-w-none lg:max-w-[700px] xl:max-w-[810px] mx-auto mt-2">
+          <turbo-frame id="messages" src="<%= conversation_messages_path(@conversation) %>">
+            Loading...
+          </turbo-frame>
+        </div>
+      </div>
+
+      <div id="right-footer" class="flex-none flex flex-col relative">
```

Comment:
I think I would also break this out into it's own partial. 

If it doesn't need an id for targeting purposes, that can be removed.

Reply:
One thing that’s nice about breaking it into a partial is that it gets a name. Then we don’t need an id. But I’ve generally not created partials unless there was some bit of HTML that was being repeated just because it’s harder to refactor the HTML structure of a page when it’s split across more files. Have you had the bias of creating partials for logical units, even if they aren’t being reused?

Reply:
I don't think I have a hard rule. When something is "layout-ish" like a two column thing like you have here, I'm way more likely to break it out.

That makes me think, perhaps this should be a dedicated layout file which gets loaded inside of the application frame for the Conversation controller?